src/runtime_src/core/tools/xbutil2/SubCmdReset.cpp

```diff
@@ -28,6 +29,33 @@ namespace po = boost::program_options;
 #include <iostream>
 
 // ----- C L A S S   M E T H O D S -------------------------------------------
+static void
+pretty_print_action_list(xrt_core::device_collection& deviceCollection, const std::string& reset)
```

Comment:
This method does "more" then what the name implies. 

It does a DRC check to determine if the option is valid AND does a pretty print.  Business rule logic should not be included in this method OR the method should be renamed.